resources/views/applicant/application_post_02.html.twig

```diff
@@ -35,6 +35,6 @@
 
     {# Modals #}
 
-        {% include "common/modal" with {"page":application_template} %}
+        {% include "common/modal" with {"page":application} %}
```

Comment:
Is this modal actually used for anything? I'm not seeing a way to open it locally.

Reply:
#372

One day, maybe.